app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -150,7 +150,10 @@ private void setupDrawerContent(NavigationView navigationView) {
 
         menu.setGroupCheckable(GROUP_ID, true, true);
 
-        menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME)).setChecked(true);
+        MenuItem selectedItem = menu.findItem(getIntent().getIntExtra(EXTRA_SELECTED_DRAWER_ITEM_ID, Const.DRAWER_HOME));
```

Comment:
Can do it but I have to go to bed now. :) I already send a beta update with these merged locally. :) 
Will do the release after all of them merged. We can do it tomorrow.